modules/core/src/main/scala/gem/config/InstrumentConfig.scala

```diff
@@ -13,6 +13,15 @@ sealed abstract class InstrumentConfig extends Product with Serializable {
       case f2: F2Config     => Some(f2.key)
       case GenericConfig(_) => None
     }
+
+  def instrument: Instrument = {
+    import gem.enum.Instrument._
+
+    this match {
```

Comment:
It's just a way to go from the specific `InstrumentConfig` back to the `Instrument` enum.  Right now we only have two `InstrumentConfig`s.  `F2Config` for Flamingos2 and `GenericConfig` for everything else.